src/claimer.py

```diff
@@ -59,6 +59,12 @@ def claim_product(api_client, recaptcha_solution):
             'dateTo': (utc_today + dt.timedelta(days=1)).isoformat()
         }
     )
+    """Traitement de l'absence d'offre"""
```

Comment:
Je ne parle pas français mon ami ;) I don't understand what's written here, but I guess it should be a comment, not a docstring (or multiline string).

Let's stick to english, as it'll be understandable by most users, I can help and proofread all the messages.

Reply:
You are right (I was handling a meeting while writing the code... two many things to think at same time). This is corrected.